tensorflow_addons/rnn/cell.py

```diff
@@ -366,3 +366,471 @@ def _create_norm_layer(self, name):
             gamma_initializer=self.norm_gamma_initializer,
             epsilon=self.norm_epsilon,
             name=name)
+
+
+@tf.keras.utils.register_keras_serializable(package='Addons')
+# class LayernormSimpleRNNCell(SimpleRNNCell, LayerNormalization):
+class LayernormSimpleRNNCell(keras.layers.SimpleRNNCell):
+    """Cell class for LayernormSimpleRNN.
+
+    Motivation:
+    - Drop-In Replacement for keras.layers.SimpleRNNCell
+    - demonstrate how to add keras.layers.LayerNormalization
+       to all RNNs by introducing the `use_layernorm` argument
+
+    References:
+    [1] Ba, Jimmy Lei, Jamie Ryan Kiros, and Geoffrey E. Hinton.
+        "Layer Normalization." ArXiv:1607.06450 [Cs, Stat],
+        July 21, 2016. http://arxiv.org/abs/1607.06450
+
+    Arguments:
+      units: Positive integer, dimensionality of the output space.
+      activation: Activation function to use.
+        Default: hyperbolic tangent (`tanh`).
+        If you pass `None`, no activation is applied
+        (ie. "linear" activation: `a(x) = x`).
+      use_bias: Boolean, (default `True`), whether the layer uses a bias
+        vector.
+      use_layernorm: Boolean, (default `True`), whether to apply layer
```

Comment:
Please remove this flag and update the logic below to always use layernorm.

Reply:
It's already changed in https://github.com/tensorflow/addons/commit/3f24525b62b0929d28eb5c3c97aa846c9d340659

With the flag `use_layernorm=False`, it's possible to unit test against the `SimpleRNN`. Without, it's not.

Reply:
https://github.com/tensorflow/addons/pull/841/commits/7a3c5f182cb94621ae81b0a9eab156b9486edf00

Reply:
done

Reply:
`use_layernorm` is gone